src/foam/nanos/logger/RepeatLogger.js

```diff
@@ -0,0 +1,224 @@
+/**
+ * @license
+ * Copyright 2018 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+
+foam.CLASS({
+  package: 'foam.nanos.logger',
+  name: 'RepeatLogger',
+  extends: 'foam.nanos.logger.ProxyLogger',
+  requires: [
+    'foam.nanos.logger.LogLevel'
+  ],
+
+  properties: [
+    {
+      class: 'Object',
+      name: 'lastUniqueObject',
+      value: null
+    },
+    {
+      class: 'Int',
+      name: 'repeatCount',
+      value: 0
+    },
+    {
+      class: 'Enum',
+      of: 'foam.nanos.logger.LogLevel',
+      name: 'lastLogLevel',
+      factory: function() { return foam.nanos.logger.LogLevel.INFO; }
+    }
+  ],
+
+  methods: [
+    {
+      name: 'newLogHandler',
+      args: [ 
+        {
+          name: 'logLevelName',
+          javaType: 'Enum'
+        },
+        {
+          name: 'setLastLogLevel',
+          javaType: 'Boolean'
+        },
+        {
+          name: 'args',
+          javaType: 'Object...'
+        },
+      ],
+      javaReturns: 'void',
+      javaCode: `
+  switch ( LogLevel.values()[logLevelName.ordinal()] ) {
```

Comment:
Can this be reduced to 
setRepeatCount(1)
setLastUniqueObject(orgs);
switch ( .. ) {
}
if (setLastLogLevel) {
setLastLogLevel(logLevelName)
}